www/docs/deploy/docker/README.mdx

```diff
@@ -63,28 +62,38 @@ If the automatic install fails, or you'd just like more control over the
 installation process, follow the steps below to install manually.
 
 1. Download the docker compose template to a local working directory:
-**For Linux**:
+   **For Linux**:
+
 ```
 curl -fsSL https://raw.githubusercontent.com/firezone/firezone/master/docker-compose.prod.yml -o docker-compose.yml
 ```
+
 **For macOS, Windows (non-production only)**:
+
 ```
 curl -fsSL https://raw.githubusercontent.com/firezone/firezone/master/docker-compose.desktop.yml -o docker-compose.yml
 ```
+
 1. Generate required secrets:
+
 ```
 docker run --rm firezone/firezone bin/gen-env > .env
 ```
-1. At a minimum, change the `ADMIN_EMAIL` and `EXTERNAL_URL` variables.
-Optionally modify other secrets as needed.
+
+1. At a minimum, change the `DEFAULT_ADMIN_EMAIL` and `EXTERNAL_URL` variables.
+   Optionally modify other secrets as needed.
 1. Migrate the database:
+
 ```
 docker compose run --rm firezone bin/migrate
 ```
+
```

Comment:
This is VSCode markdown formatter, should I roll it back?

Reply:
No worries, if it passes CI it should be fine